lib/stdlib.cf

```diff
@@ -71,6 +71,9 @@ bundle common stdlib_common
       "input[cfe_internal_hub]"           string => "$(this.promise_dirname)/cfe_internal_hub.cf";
       "input[cfengine_enterprise_hub_ha]" string => "$(this.promise_dirname)/cfengine_enterprise_hub_ha.cf";
 
+      # jUnit reports
+      "input[junit]"                      string => "$(this.promise_dirname)/junit.cf";
```

Comment:
We could make this optional if the `junit` class is defined but that seemed unnecessary since they are all agent bundles that do nothing by default.